test/unit/forge/db/controllers/ProjectTemplate_spec.js

```diff
@@ -17,6 +17,7 @@ describe('Project Template controller', function () {
     describe('Validate Settings', function () {
         it('only allows settings from the known list', async function () {
             const result = app.db.controllers.ProjectTemplate.validateSettings({
+                credentialSecret: 'aaaaaaaabbbbbbbbccccccccdddddddd',
```

Comment:
This isn't quite the right set of tests - this is testing providing the credentialSecret via Template settings - which is different to being provided via ProjectSettings.